cli/pcluster/update/update_command.py

```diff
@@ -179,7 +179,7 @@ def _restore_cfn_only_params(cfn_boto3_client, args, cfn_params, stack_name, tar
 
     scheduler = cluster_section.get_param_value("scheduler")
     # Autofill DesiredSize cfn param
-    if not args.reset_desired:
+    if not args.reset_desired and scheduler != "slurm":
```

Comment:
I see this condition on the slurm scheduler, meant to check if we are in a HIT cluster, is used in many places; I'm wondering this condition could be lost among all other scheduler checks which can be there for different reasons. What about being more explicit and adding a global function like ``is_hit_cluster()`` that can currently just do the same test on the scheduler but would isolate the rest of the code from this information (and will be also ready to work in the case other schedulers will be covered by HIT support)?

Reply:
👍🏻